invenio_communities/communities/services/service.py

```diff
@@ -714,6 +714,25 @@ def on_relation_update(self, identity, record_type, records_info, notif_time):
 
         return True
 
+    @unit_of_work()
+    def bulk_update_parent(self, identity, comm_slugs, new_parent, uow=None):
```

Comment:
```suggestion
    def bulk_update_parent(self, identity, community_ids, parent, uow=None):
```

minor: slugs and UUIDs are resolved uniformly in communities, so it's ok to be "generic" about it :)

Also some renaming to simplify.